azurerm/resource_arm_log_profile.go

```diff
@@ -0,0 +1,264 @@
+package azurerm
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2018-03-01/insights"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmLogProfile() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmLogProfileCreateOrUpdate,
+		Read:   resourceArmLogProfileRead,
+		Update: resourceArmLogProfileCreateOrUpdate,
+		Delete: resourceArmLogProfileDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			"storage_account_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: azure.ValidateResourceIDOrEmpty,
+			},
+			"service_bus_rule_id": {
```

Comment:
can we make this `servicebus_rule_id` to match the other resources/Azure branding?